### DISCUSSION TITLE
documentation sanic ASGI

### DISCUSSION THREAD

**Discussion by raphaelauv** [fastapi/fastapi#7748] [Questions]
### Description

Sanic framwork now support ASGI 
https://sanic.readthedocs.io/en/latest/sanic/deploying.html#running-via-asgi

documentation should be updated https://fastapi.tiangolo.com/alternatives/#sanic

Thank you all !

**Comment by tiangolo** [comment]
Thanks for the report and the PR @raphaelauv !
